src/components/header/Backtrace.css

```diff
@@ -0,0 +1,25 @@
+.backtrace {
+  margin: 5px;
+  padding: 0;
+  list-style: none;
+  display: block;
+  max-height: 25em;
+  overflow: auto;
+}
+
+.backtraceStackFrame {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: block;
```

Comment:
same :)

Reply:
This one actually has a use: `li` is `display: list-item` by default. The `list-style: none` declaration makes `display: list-item` act very similarly (or maybe even identically?) to `display: block`, so strictly speaking, you'd only need either `display: block` or `list-style: none`, but I always specify both out of habit.